.github/workflows/kubeflow-pipelines-integration-v2.yml

```diff
@@ -0,0 +1,38 @@
+name: Kubeflow Pipelines V2 integration Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - '.github/workflows/kubeflow-pipelines-integration-v2.yml'
+      - 'samples'
+      - 'core'
+      - 'backend'
```

Comment:
`backend` is not included in https://github.com/GoogleCloudPlatform/oss-test-infra/blob/68230effbdc9d59429b205475325c3212ac887d7/prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml#L61.
Do you really think it should be included here?

Reply:
Sure!